keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -199,18 +205,117 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
 
 * **What specific metrics should inform a rollback?**
-  N/A.
+Check the following indicators to determine if there are any exceptions:
+  - pod_preemption_victims
+  - total_preemption_attempts
+  - scheduling_algorithm_preemption_evaluation_seconds
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  N/A.
+  Manually tested successfully.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features?**
   N/A.
 
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+N/A. This is not a feature that workloads use directly
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events
+  - Event Reason:
+- [ ] API .status
+  - Condition name:
+  - Other field:
+- [x] Other (treat as last resort)
+  - Details: Check if pods with preemptionPolicy set to Never can preempt other low-priority pods when the cluster resources cannot be met.  
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+N/A
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [x] Metrics
+  - Metric name: preemption_victims
+  - [Optional] Aggregation method:
+  - Components exposing the metric: kube-scheduler
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature? 
+No.
```

Comment:
The questions about knowing whether the feature is in use don't have very useful answers. New metrics or labels might help. But maybe they are not realistic, so you can describe that here.

Reply:
Done